lib/Udp/V5MiLightUdpServer.cpp

```diff
@@ -64,6 +81,51 @@ void V5MiLightUdpServer::handleCommand(uint8_t command, uint8_t commandArg) {
         );
         break;
 
+      case UDP_RGBW_GROUP_ALL_NIGHT:
```

Comment:
Would we want to `client->prepare(&FUT096Config, deviceId, 0)` here (and do the same thing for the other groups in the other cases)?  Otherwise we'll send the white mode command to the last group a packet was received for.

Reply:
Can you explain this?

Reply:
`client->enableNightMode()` will enable night mode for the group seen in the last command.  So if you turned on group 2, and then enabled night mode for group 1, it'd enable night mode for group 2.

The fix would be to add the `client->prepare` statement for the appropriate group.  This call tells the client to format and send packets for the specified radio, device, and group combo.

Reply:
Ah I see... OK, thats why my PHP API always sends OFF before engaging night mode. :)
Worked our a fix for that based on your idea.
https://github.com/bombcheck/esp8266_milight_hub/commit/e3f3ca993036d486f42061fdfd0fc25b00aaea02